staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -436,13 +503,17 @@ func (o *AuditLogOptions) getWriter() io.Writer {
 	return w
 }
 
-func (o *AuditLogOptions) applyTo(c *server.Config) error {
+func (o *AuditLogOptions) newBackend(checker policy.Checker) audit.Backend {
 	if w := o.getWriter(); w != nil {
+		if checker == nil {
```

Comment:
seems like this logic should just be done at a higher level than this function. Passing a "don't do work" argument isn't a good api.